src/tools/dashboard/components/Row.tsx

```diff
@@ -183,11 +182,16 @@ const NiceStatus = ({ job, jobLink }: NiceStatusProps) => {
           const hitText = pluralise('hit', actualHits);
           return (
             <>
-              {status}{' '}
+              {actualHits === 0 ? (
+                <span>Completed</span>
+              ) : (
+                // eslint-disable-next-line uniprot-website/use-config-location
+                <Link to={jobLink}>Completed</Link>
+              )}{' '}
               <span
                 title={`${actualHits} ${hitText} results found instead of the requested ${expectedHits}`}
               >
-                ({job.data.hits} {hitText})
+                ({actualHits ? `${actualHits} ${hitText}` : 'No results found'})
```

Comment:
```suggestion
                ({actualHits ? `${actualHits} ${hitText}` : 'no results found'})
```
to yield `Completed (no results found)`. I'm being pedantic here but I just find only having the `Completed` capitalized a little easier on the eyes - maybe because i'm reading it like a sentence.

Reply:
I see, yep, looks better actually